packages/cli/src/utils/environment-tasks.ts

```diff
@@ -98,19 +98,29 @@ export const start: ListrTask<IStart> = {
     {
       title: 'Starting the Engine',
       skip: async ctx => await isRunning(ctx.endpoint) || (await listContainers({ label: [engineLabel] })).length > 0,
-      task: ctx => {
-        write(ctx.configDir, {
-          account: {
-            mnemonic: ctx.mnemonic
-          }
-        })
-        return createContainer(`${image}:${ctx.version}`, ctx.configDir)
-      }
+      task: ctx => createContainer(`${image}:${ctx.version}`, ctx.configDir)
     },
     {
       title: 'Waiting for the Engine to be ready',
       skip: ctx => isRunning(ctx.endpoint),
       task: async ctx => waitToBeReady(ctx.endpoint)
+    },
+    {
+      title: 'Ensure balance',
```

Comment:
This might be replaced by generating the genesis with some token for all public addresses in the authorized pubkeys in the engine but it doesn't make much sense